packages/Sortable/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@paprika/sortable",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
I don't think we want to update versions manually in PRs. It should be handled by lerna when you publish (at which point you determine whether it's a major/minor/patch bump). cc @nahumzs

Reply:
How about the devdependencies in `paprika/package.json` file? Do we need to manually update it to keep it updated?

Reply:
Good question, I suspect it _should_ be part of our publishing procedure though.

Reply:
Lerna allowed the option of manually bumping the version vs automatically handle. Maybe we should discuss this a little further, but right now you don't need to bump the version mike

Reply:
Yep, reverted in a more recent commit.